recipe/patch_yaml/ray-grpcio.yaml

```diff
@@ -0,0 +1,19 @@
+# Prevent ray2.9.0 and grpcio 1.58 (https://github.com/conda-forge/ray-packages-feedstock/issues/136)
+if:
+  name: ray-default
+  version_le: "2.9.0"
+then:
+  - replace_depends:
+      old: grpcio
+      new: grpcio !=1.58
```

Comment:
```suggestion
      new: grpcio =1.57
```

Reply:
see above, we really don't want to do this as it makes ray not work with most of the things you want to use it with

Reply:
I’ve seen and participated in the above discussion by asking relevant questions in the feedstock PR.

You asked why this is taking too long to merge in. Making patches that just have to be patched again later is not a sustainable strategy. 

I personally don’t have enough knowledge of the details of grpcio to guarantee that 1.57 will work with 1.60 for example. If you want to allow 1.57 and 1.59 I might consider it. But following the grpcio pinnings, it doesn’t seem safe to leave the upper bound of grpcio unbound. 

If an other core member agrees with your strategy and disagrees with mine they are entirely free to merge this. I am just applying conda-forge best practices to help review this request and get your package running.

please don’t outright dismiss the real concerns I raised in the feedstock PR.

Reply:
not dismissing, but hard pinning to a specific version will immediately break a lot of people who are otherwise fine. And thats not great from a best practices or community standpoint

Reply:
> You asked why this is taking too long to merge in.

I did not, not sure where you got that from...I am a maintainer trying to get the right outcome

Reply:
I think a pin to both 1.57 and 1.59 might be acceptable. However. I haven’t tested.

Reply:
@apmorton has already shown 1.56 works as well. so the conservative approach is to disable known-non-working versions, which I think is least disruptive. disabling all excerpt for 1 known-working version seems to be overly disruptive.

Reply:
so long as some upper bound on the version exists.